tests/test_planet_routes.py

```diff
@@ -0,0 +1,96 @@
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet_with_no_records_returns_404(client):
+    #Act
+    response = client.get("/planets/1")
+
+    #Assert
+    assert response.status_code == 404
+
+def test_get_one_planet_with_records_successful(client, two_saved_planets):
+    #Act
+    response = client.get("/planets/1")
```

Comment:
If we returned the record list from the fixture, then we could write this line as
```py
    response = client.get(f"/planets/{two_saved_planets[0].id}")
```